resident/resident-service/src/main/java/io/mosip/resident/validator/DocumentValidator.java

```diff
@@ -123,8 +125,11 @@ public void validateDocumentId(String documentId) {
 	public void validateFileName(MultipartFile file) {
 		String extension = Objects.requireNonNull(FilenameUtils.getExtension(file.getOriginalFilename())).toLowerCase();
 		String extensionProperty = Objects.requireNonNull(env.getProperty(ALLOWED_FILE_TYPE)).toLowerCase();
-		if(!extensionProperty.contains(Objects.requireNonNull(extension))){
+		if (!extensionProperty.contains(Objects.requireNonNull(extension))) {
 			throw new InvalidInputException(ResidentConstants.FILE_NAME);
 		}
+		if (file.getSize() > Integer.parseInt(env.getProperty(FILE_SIZE))) {
```

Comment:
Change this to property loaded with @Value

Reply:
Changes done accordingly